### PR TITLE
K8sdocs: add warning about juju/lxd bug

### PR DESCRIPTION
**Pull request by evilnick**
## Done

- Adds a note on upgrade pages re Juju bug with LXD
-  Adds troubleshooting section for fixing LXD profiles

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

### CODE REVIEW

File: templates/kubernetes/docs/1.20/upgrading.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-wrapper_template: "templates/docs/markdown.html"
+wrapper_template: "kubernetes/docs/base_docs.html"
```
Comment (anthonydillon) [inline templates/kubernetes/docs/1.20/upgrading.md:2]: Can you revert this change please

File: templates/kubernetes/docs/upgrade-notes.md [hunk 2]
```diff
@@ -19,6 +19,11 @@ The notes are organised according to the upgrade path below, but also be aware t
 upgrade that spans more than one minor version may need to beware of notes in
 any of the intervening steps.
 
+## Upgrades to all versions deployed to Juju's `localhost` LXD based cloud
+
+There is a known issue ([https://bugs.launchpad.net/juju/+bug/1904619](https://bugs.launchpad.net/juju/+bug/1904619))
+with container profiles not surviving an upgrade in clouds running on LXD. If your container-based applications fail to work properly after an upgrade, please see this [topic on the troubleshooting page](/kubernetes/docs/troubleshooting#lxd)
```
Comment (anthonydillon) [inline templates/kubernetes/docs/upgrade-notes.md:25]: Wouldn't it be better to link directly to the title? `/kubernetes/docs/troubleshooting#charms-deployed-to-lxd-containers-fail-after-upgradereboot`

File: templates/kubernetes/docs/troubleshooting.md [hunk 3]
```diff
@@ -115,6 +115,88 @@ Running the `juju-crashdump` script will generate a tarball of debug information
 
 ## Common Problems
 
+<a id="lxd"> </a>
```
Comment (anthonydillon) [inline templates/kubernetes/docs/troubleshooting.md:118]: All titles get id's so you could remove this `id` and target `#charms-deployed-to-lxd-containers-fail-after-upgradereboot`

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Few comments in the code.

File: templates/kubernetes/docs/1.20/upgrading.md (hunk 1 above)
Comment (evilnick) [reply templates/kubernetes/docs/1.20/upgrading.md:2]: oops, didn't notice this :+1:

File: templates/kubernetes/docs/upgrade-notes.md (hunk 2 above)
Comment (evilnick) [reply templates/kubernetes/docs/upgrade-notes.md]: done

**Comment by evilnick** [comment]
@anthonydillon thanks for spotting that template bit! should be all fixed up now

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:
